src/views/forms/TradeForm/TradeSizeInputs.tsx

```diff
@@ -1,22 +1,24 @@
-import { useCallback, useEffect, useState } from 'react';
+import { useCallback, useEffect } from 'react';
 
+import _ from 'lodash';
```

Comment:
can you use named imports in case we get tree-shaking? 
```suggestion
import { debounce } from 'lodash';
```